packages/nextjs/hooks/scaffold-eth/useScaffoldReadContract.ts

```diff
@@ -29,7 +31,7 @@ export const useScaffoldContractRead = <
   const { data: deployedContract } = useDeployedContractInfo(contractName);
   const { targetNetwork } = useTargetNetwork();
 
-  return useContractRead({
+  return useReadContract({
```

Comment:
Should we remove `watch` and `enabled` from here? Ignore if I'm totally wrong pls!

Reply:
Ohh yes !! TYSM !!!! Actually there was bug which I didn't noticed , we were passing `enabled` option directly but instead it should have been passed in `query` object, updated it at https://github.com/scaffold-eth/scaffold-eth-2/pull/700/commits/388bc9c15b666ce0bbf08b102e532cc63b3937c8

Also btw, even though wagmi have removed `watch` option, we will be allowing it to be passed to our hook (default to `true` similar to SE-2 wagmi v1) update at https://github.com/scaffold-eth/scaffold-eth-2/pull/700/commits/4462752e0fd4aa9ee2cc9f7b99b7da094102bb9d

So I think we should also add `watch` option here https://docs.scaffoldeth.io/hooks/useScaffoldContractRead#configuration with default value as `true`  and maybe desc like : "Watches and refreshes data for new blocks"